jupyterlab/tests/test_commands.py

```diff
@@ -0,0 +1,101 @@
+"""Test yarn registry replacement"""
+
+# Copyright (c) Jupyter Development Team.
+# Distributed under the terms of the Modified BSD License.
+import logging
+import os
+from os.path import join as pjoin
+from tempfile import TemporaryDirectory
+from unittest import TestCase
+from unittest.mock import patch
+
+from jupyter_core import paths
+
+from jupyterlab import commands
+
+
+class Test_AppHandler(TestCase):
```

Comment:
The name of this class and of the file is slightly confusing, since there are a lot of tests for the apphandler commands in `test_jupyterlab.py`. I would either:
- move these tests there
- rename this file to something more appropriate
- move tests from `test_jupyterlab.py` here as appropriate

Reply:
I went for renaming the file.